docs/FAQ.md

```diff
@@ -288,6 +289,15 @@ No.
 
 For a refresher on Go's recommended workspace organization, see the ["How To Write Go Code"](https://golang.org/doc/code.html) article in the Go docs. Organizing your code this way gives you a unique import path for every package.
 
+## How do I make `dep` resolve dependencies from my `GOPATH`?
+
+By default, `dep` runs in network mode and would download all the dependencies
+from the network while initializing (`init`) and restoring (`ensure`). `init`
```

Comment:
hmm, i think it might be misleading to even mention `dep ensure`, or "network mode." IMO the key knowledge the reader needs to come away with from this question is:

1. there's a flag on `dep init`
2. `dep ensure` does not ever work with projects on `GOPATH`